src/main/java/org/sonar/plugins/stash/issue/collector/StashCollector.java

```diff
@@ -51,7 +54,10 @@ public static StashComment extractComment(JsonObject jsonComment, String path, L
     JsonObject jsonAuthor = (JsonObject)jsonComment.get(AUTHOR);
     StashUser stashUser = extractUser(jsonAuthor);
 
-    return new StashComment(id, message, path, line, stashUser, version);
+    StashComment result = new StashComment(id, message, path, line, stashUser, version);
+    // FIXME do this at some central place

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1134)